cors/cors.go

```diff
@@ -0,0 +1,127 @@
+package cors
+
+import (
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+)
+
+type OriginChecker interface {
+	Options(events.APIGatewayV2HTTPRequest) events.APIGatewayV2HTTPResponse
+	GetCorsHeaders(request events.APIGatewayV2HTTPRequest) (map[string]string, bool)
+}
+
+// Domains contains the allowed domains and the accepted methods for each one
+type Domains struct {
+	Allowed map[string]map[string]bool
+}
+
+// NewDomains returns the default Domains
```

Comment:
I'm surprised the linter didn't catch this:
```suggestion
// NewOriginChecker returns the default Domains
```